wordpress-mobile/WordPress-iOS
# Adds a "Confetti" view to be used anywhere in the app.

This PR also adds a feature flag for Milestone Notifications

Fixes #NA

**Note**
At the moment, the confetti view is hooked to the existing achievement system (that is why it's still in draft status)

To test:
- Before building
    - set the `milestoneNotifications` feature flag to `true`
    - in `NotificationsDetailViewController`, edit the `isViewMilestone` property to match an existing notification type, e.g. a likes or "first post" milestone. In this case,  in that property, you can replace `note.type == "view_milestone"` with `note.type == "like_milestone_achievement"` or with `note.type == "post_milestone_achievement"` (or both, if you want to test it on more notifications).



- Build and run
    - log in with an account that have existing achievement notifications (e.g. likes), or trigger an achievement notification from another source
    - Go to "Notifications" and open the achievement notifications, and make sure the confetti view is displayed (see example below)


<p align=center>
<img height=500 src=https://user-images.githubusercontent.com/34376330/108410550-2b258880-71ed-11eb-9fa6-a5b795389dfa.gif>
</p>


PR submission checklist:
